executor/api/util/utils.go

```diff
@@ -141,3 +141,26 @@ func GetEnvAsBool(key string, fallback bool) bool {
 
 	return fallback
 }
+
+type SslKakfa struct {
+	KafkaSslClientCertFile string
```

Comment:
:upside_down_face: Minor point, but mentioning for good style - given the struct is called `SslKafka` and it's not being embedded anywhere, there's no need to have the `KafkaSsl` prefix for each field as it makes it a bit verbose to read